cmis/src/main/java/nl/nn/adapterframework/extensions/cmis/CmisSender.java

```diff
@@ -591,12 +589,10 @@ private void processProperties(Element propertiesElement, Map<String, Object> pr
 					if(AppConstants.getInstance().getBoolean("cmissender.processproperties.legacydateformat", false)) {
 						formatStringAttr = "yyyy-MM-dd HH:mm:ss";
 					}
-					DateFormat df = new SimpleDateFormat(formatStringAttr);
 					GregorianCalendar calendar = new GregorianCalendar();
 					try {
-						Date date = df.parse(property);
-						calendar.setTime(date);
-					} catch (ParseException e) {
+						calendar.setTimeInMillis(DateUtils.parseToInstant(property, formatStringAttr).toEpochMilli());
```

Comment:
Ik weet niet of dit nou zo mooi is.. 
Een alternatief zou zijn `GregorianCalendar.from(ZonedDateTime.ofInstant(instant));`
Al weet ik niet of dat veel beter is. Misschien dat je dit soort dingen ergens in een util klasse kan stoppen.. (hint hint)